linux_os/guide/system/software/integrity/disable_prelink/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: ol7,rhel7
+prodtype: rhcos4,ol7,rhel7,wrlinux1019
```

Comment:
`disable_prelink` is not enabled for `rhel8`, so I wonder what requires that on `rhcos4`.

Reply:
So the answer is that `grub2_enable_fips_mode` needs it. Sounds legit :-)